client/ayon_shotgrid/shotgrid_module.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from openpype.modules import (
+    OpenPypeModule,
+    ITrayModule,
+    IPluginPaths,
+)
+
+SHOTGRID_MODULE_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+class ShotgridModule(OpenPypeModule, ITrayModule, IPluginPaths):
+    name = "shotgrid"
+    enabled = False
+    project_id = None
+    tray_wrapper = None
+
+    def initialize(self, modules_settings):
+        self.module_settings = modules_settings.get(self.name, dict())
```

Comment:
It is much better to not store whole settings but only keys that should be available on addon.